.travis.yml

```diff
@@ -0,0 +1,31 @@
+sudo: false
+dist: trusty
+language: erlang
+otp_release:
+  - 20.0.1
+  - 19.3.6.1
+  - 18.3
+addons:
+  apt:
+    sources:
+      - debian-sid
+    packages:
+      ## GNU Make 4.  GNU Make 3.81 as [in Ubuntu 14.04
+      ## "trusty"](https://packages.ubuntu.com/trusty/make) is not
+      ## sufficient, because of error on `triq` test dep when running
+      ## `make tests`.  Also erlang.mk reports the following warning:
+      ## "erlang.mk:24: Please upgrade to GNU Make 4 or later:
+      ## https://erlang.mk/guide/installation.html".
+      - make
+before_install:
+  - make -v
+script:
+  - make ${JOB:?}
+matrix:
+  include:
+    - language: erlang
```

Comment:
[Travis WebLint](http://lint.travis-ci.org/) reports: “in matrix.include section: unexpected key language, dropping”.

Reply:
I have not time for running trials on this today. The main aim is running job edoc only with one Erlang version - I fear removing keys erlang and/or otp_release may make the job run for all OTP releases (hence 3 jobs). What do you suggest?

Reply:
I’ve pushed a test branch `travis-test` with the `language` line removed and it looks like there is still only one `edoc` job, so presumably we can safely remove this one line.